tests/functional/m/modified_iterating.py

```diff
@@ -1,7 +1,8 @@
 """Tests for iterating-modified messages"""
-# pylint: disable=not-callable,unnecessary-comprehension,too-few-public-methods
+# pylint: disable=not-callable,unnecessary-comprehension,too-few-public-methods, missing-class-docstring, missing-function-docstring
```

Comment:
```suggestion
# pylint: disable=not-callable,unnecessary-comprehension,too-few-public-methods,missing-class-docstring,missing-function-docstring
```

I don't have a preference for white spaces after comma's in a disable comment, but let's follow the pre-existing style.